### PR TITLE
OE-91: Added an "external reference" field to trials

### PR DESCRIPTION
**Pull request by Vegemash**

### CODE REVIEW

File: migrations/m170711_024649_add_external_ref_column.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+class m170711_024649_add_external_ref_column extends OEMigration
+{
+    // Use safeUp/safeDown to do migration with transaction
+    public function safeUp()
```
Comment (apigram) [inline migrations/m170711_024649_add_external_ref_column.php]: up() and down() should be sufficient here as there is no DML that requires transactional logic.

File: views/trial/view.php [hunk 2]
```diff
@@ -57,7 +57,15 @@
           </div>
         <?php endif; ?>
 
-        <?php if ($hasManagePermissions): ?>
+        <?php if (strlen($model->external_reference) > 0): ?>
```
Comment (apigram) [inline views/trial/view.php]: According to PSR-2, this can simply be represented as $model->external_Reference !== ''

File: views/trial/view.php [hunk 3]
```diff
@@ -57,7 +57,15 @@
           </div>
         <?php endif; ?>
 
-        <?php if ($hasManagePermissions): ?>
```
Comment (marshl) [inline views/trial/view.php]: Oh, um, you should be using this lone, not the >= one

File: migrations/m170711_024649_add_external_ref_column.php (hunk 1 above)
Comment (marshl) [inline migrations/m170711_024649_add_external_ref_column.php]: Fun fact: DDL is transaction in Postgres, so safeUp() and safeDown() are still recommended

File: views/trial/view.php (hunk 2 above)
Comment (marshl) [inline views/trial/view.php]: What if it is null?

**Pull request merged**